eppo_client/assignment_logger.py

```diff
@@ -8,3 +8,6 @@ class AssignmentLogger(BaseModel):
 
     def log_assignment(self, assignment_event: Dict):
         pass
+
+    def log_bandit_action(self, bandit_event: Dict):
+        pass
```

Comment:
@aarsilv in particular: added the `log_bandit_action` method to the assignment logger for now, it seems like the simplest solution but happy to adjust

Reply:
I think that's fine! I don't know python well but the main goal will be that only customers using bandits will need to implement / worry about this